db/Dockerfile

```diff
@@ -9,11 +9,12 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ADD make_db.sh /docker-entrypoint-initdb.d/
-ADD setup-wale.sh /docker-entrypoint-initdb.d/
-COPY docker-entrypoint1.sh /
-RUN chmod +x /docker-entrypoint1.sh
```

Comment:
you don't see it, but it is executable. Clone the repo on your machine, and just `ls -lat` to see the mode.
This saves one line of code to maintain :)